x-pack/plugins/watcher/common/constants/watch_tabs.ts

```diff
@@ -1,30 +0,0 @@
-/*
```

Comment:
since this is only being used in one place, i removed it from the constants directory